fhireval/data/cmg-research-study.json

```diff
@@ -0,0 +1,163 @@
+{
+	"groupname": "CMG-Examples",
+	"resources": {
+	  "ResearchStudy": [
+      {
+        "resourceType": "ResearchStudy",
+        "identifier": [ {
+          "system": "urn:ncpi:id:unique-string",
+          "value": "FHIR-CMG-TEST"
+        } ],
+        "title": "FHIR-CMG-TEST",
+        "status": "completed"
+      }
+    ],
+    "ResearchSubject": [
+      {
+        "resourceType": "ResearchSubject",
+        "identifier": [ {
+          "system": "urn:ncpi:id:unique-string",
+          "value": "ct.patient09"
+        }, {
+          "system": "urn:ncpi:unique-string",
+          "value": "FHIR-CMG-TEST|ct.patient09"
+        } ],
+        "status": "on-study",
+        "study": {
+          "reference": "ResearchStudy?identifier=FHIR-CMG-TEST"
+        },
+        "individual": {
+          "reference": "Patient?identifier=ct.patient09"
```

Comment:
I am not sure if referencing against an identifier query parameter is a secure way. Multiple studies can have duplicate patient IDs because these IDs have unique meanings in each study (e.g. Study A can have Patient `1` while it is also possible to Study B). `system` is a context of `value` being unique, so I would argue referencing a logical ID should always be preferred. Plus, referencing a bundle of entries is not supported in FHIR as far as I know.

Reply:
These data files are actually modified before being submitted to the server and those references are overwritten with references to appropriate IDs. I did change it so that they were more consistent.